README.md

```diff
@@ -1,48 +1,121 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Background
 
-## Getting Started
+### Admin portal
 
-First, run the development server:
+This project is used to interact with JSON specifications via the FIA API, which if running locally needs to be backed by a local FIA_API its postgress database. Requires docker if building docker containers locally.
 
-```bash
-npm run dev
-# or
-yarn dev
-# or
-pnpm dev
-# or
-bun dev
-```
+### Tech used
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+- **Next JS** - The web development framework, the backbone of the project which provides various features and functionality (SSG, SSR, etc.)
+- **MUI** - For consistent UI aesthetics
+- **Monaco Code Editor** - For the JSON specification editing feature
+- **Cypress** - For running tests
+- **Docker** - For manually building and pushing containers to registry
 
-You can start editing the page by modifying `app/page.tsx`. The page auto-updates as you edit the file.
+## Getting Started (local install)
 
-This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
+Clone the repository, and navigate to the root directory of the newly cloned repo
 
-## Learn More
+`git clone https://github.com/fiaisis/admin-portal.git`
 
-To learn more about Next.js, take a look at the following resources:
+`yarn install` or `yarn --frozen-lockfile`- Installs the required packages
 
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+`yarn build`- To build the project
 
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
+Then you can run the NextJS server using two methods. As a development server (this comes with hot reloading of the server after code changes). Or as a local environment that mimics a production server.
```

Comment:
This part is slightly misleading as you do not need to run `yarn build` if you are going to start the dev server with `yarn dev`

Reply:
You are correct that build isn't needed for dev. 

However if you are constantly switching between `yarn dev` and `yarn start`, you may end up with this order of commands which causes `yarn start` to fail...

`yarn build` ->yarn dev -> `yarn start`

Granted this could be avoided if you always run `yarn build` before `yarn start`.